### PR TITLE
feat: add xball

### PR DESCRIPTION
**Pull request by liushanyang12138**
xball: Simulate bouncing balls in a window

Log: add xball
Issue:
deepin-community/sig-deepin-sysdev-team#274

### CODE REVIEW

**Comment by liushanyang12138** [comment]
@Zeno-sole @chenchongbiao 请老师审查

File: .gitignore
Comment (Zeno-sole) [inline .gitignore:1]: 存在.o等中间文件

Comment (liushanyang12138) [reply .gitignore:1]: 已修改，obs已通过

**Pull request merged**